core/src/test/java/org/aerogear/mobile/core/integration/MetricsServiceIntegrationTest.java

```diff
@@ -7,13 +7,15 @@
 
 import org.junit.Before;
 import org.junit.Test;
+import org.junit.experimental.categories.Category;
```

Comment:
It's nice to see that category annotation is experimental but the IntegrationTest category itself is not LOL

Reply:
Yes, `@Category` is junit class whereas `UnitTest` and `IntegrationTest` are our custom interfaces.